examples/lb3-application/src/__tests__/acceptance/home-page.acceptance.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/example-lb3-application
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Client} from '@loopback/testlab';
+import {CoffeeShopApplication} from '../../application';
+import {setupApplication} from './test-helper';
+
+describe('HomePage', () => {
+  let app: CoffeeShopApplication;
+  let client: Client;
+
+  before('setupApplication', async () => {
+    ({app, client} = await setupApplication());
+  });
+
+  after(async () => {
+    await app.stop();
+  });
+
+  it('exposes a default home page', async () => {
```

Comment:
Have you considered adding a test to verify LB3 home page `/lb3-index.html` too?